### PR TITLE
Add profiler quickstart content.

### PR DESCRIPTION
**Pull request by aalexand**

### CODE REVIEW

File: profiler/profiler_quickstart/busybench.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	}
+}
+
+func load() {
+	for i := 0; i < (1 << 20); i++ {
+	}
+}
+
+func main() {
+	runtime.GOMAXPROCS(8)
```
Comment (broady) [inline profiler/profiler_quickstart/busybench.go]: Why 8? Default seems fine to me.

If you disagree, please add a comment to the sample explaining why.

File: profiler/profiler_quickstart/busybench.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	err := profiler.Start(
+		profiler.Config{
+			Service:              "hello-profiler",
+			NoHeapProfiling:      true,
+			NoGoroutineProfiling: true,
+			DebugLogging:         true,
+		})
```
Comment (broady) [inline profiler/profiler_quickstart/busybench.go]: Just a nit, but if I were writing this code, I'd probably put the profiler.Config on the same line as the call to profiler.Start.

**Comment by aalexand** [comment]
@broady PTAL

**Comment by aalexand** [comment]
@broady you probably wanted to rebase. Merging is rarely good answer with PRs.

**Comment by aalexand** [comment]
I rebased and force-pushed.

**Comment by broady** [comment]
I did mean to merge. We squash commits in this repo.

**Comment by aalexand** [comment]
Got it. It's just when someone else does a merge googlebot complains about commits from multiple authors in the same PR.

**Pull request merged**